src/app/containers/ProfilePageWrapper/index.tsx

```diff
@@ -1,20 +1,70 @@
 import classNames from "classnames/bind";
 import { Typography, Pagination, LinearProgress } from "@mui/material";
-import { useState, useCallback, useMemo } from "react";
+import Tabs from "@mui/material/Tabs";
+import Tab from "@mui/material/Tab";
+import Box from "@mui/material/Box";
+import React, { useState, useCallback, useMemo } from "react";
 
 import styles from "./ProfilePageWrapper.module.scss";
 import { BreadcrumbsCustomization } from "app/components/BreadcrumbsCustomization";
 import { ContentItem } from "app/components/ContentItem";
 import { useGetDocumentsByOwner } from "queries/document";
-import { DEFAULT_PAGINATION } from "utils/constants";
-
 import { useGetExamsByOwner } from "queries/exam";
+import {
+  useGetUserExamsCompletedByOwner,
+  useGetUserExamsNotCompletedByOwner,
+} from "queries/userExam";
+
+import { DEFAULT_PAGINATION } from "utils/constants";
 
 const cx = classNames.bind(styles);
 
+interface TabPanelProps {
+  children?: React.ReactNode;
+  index: number;
+  value: number;
+}
+
+const TabPanel = (props: TabPanelProps) => {
+  const { children, value, index, ...other } = props;
+  return (
+    <div
+      role="tabpanel"
+      hidden={value !== index}
+      id={`simple-tabpanel-${index}`}
+      aria-labelledby={`simple-tab-${index}`}
+      {...other}
+    >
+      {value === index && (
+        <Box sx={{ p: 3 }}>
+          <Typography>{children}</Typography>
+        </Box>
+      )}
+    </div>
+  );
+};
+
+const allyProps = (index: number) => {

```

Comment:
What is this? And why you don't use useCallback?

Reply:
Instead of having to write attribute: "simple-tab" , "aria-controls","simple-tabpanel" for each Tab , I create this allyProps function and return the above attributes  and i think it doesn't make the component rerender at all, no need to useCallback

Reply:
no